back/engines/free/email_campaigns/app/serializers/email_campaigns/web_api/v1/campaign_serializer.rb

```diff
@@ -22,16 +22,11 @@ class WebApi::V1::CampaignSerializer < ::WebApi::V1::BaseSerializer
     attribute :schedule_multiloc, if: proc { |object|
       schedulable? object
     } do |object|
-      # Temporary fix until CL2-3052 is solved
```

Comment:
The ticket [cl2-3052](https://citizenlab.atlassian.net/browse/CL2-3052) referenced here seems to refer to a problem with fallback languages.

Manual testing shows the list still works in English, Dutch & French, but maybe we should test more/all languages to be sure.